### DISCUSSION TITLE
how to move blog to the dropdown menu section

### DISCUSSION THREAD

**Discussion by EchoRLiu** [alshedivat/al-folio#1189] [Q&A]
Hi! I am trying to move the blog from the main nav bar to the dropdown menu but not sure how to do that. I tried to change things in the header but only managed to move it to a different part on the bar.

**Comment by george-gca** [comment]
Previously this was not easily possible, but not it has been implemented.

**Reply by gsuarezr** [reply]
how is it done?

**Reply by george-gca** [reply]
It is already done in the [template example](https://alshedivat.github.io/al-folio/). Just add it to the [dropdown](https://github.com/alshedivat/al-folio/blob/master/_pages/dropdown.md).
